common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -191,6 +191,12 @@ public class RssBaseConf extends RssConf {
       .defaultValue(60L)
       .withDescription("The kerberos authentication relogin interval. unit: sec");
 
+  public static final ConfigOption<String> RSS_SERVER_IP = ConfigOptions
```

Comment:
> Could we reuse this environment value? [6937631](https://github.com/apache/incubator-uniffle/commit/6937631876052425b8d808d26caf78c79b24536a)

Yes, but using environment variables may cause some problems if we create mutil ShuffleServer in ut, should we reuse?

Reply:
You can see the pr's test case, it's not a big problem for ut. We just need to set the environment variable for one server and start the server instead of starting all the servers.